db/benchmarks/benchmarkMongo.js

```diff
@@ -0,0 +1,122 @@
+const MongoClient = require('mongodb').MongoClient;
+
+const url = 'mongodb://localhost:27017';
+const dbName = 'wegot';
+
+MongoClient.connect(url, (err, client) => {
+  if (err) {
+    console.error(err);
+  }
+
+  const db = client.db(dbName);
+  const collection = db.collection('inheritedrestaurants');
+
+  const findDocForId = async(id) => {
+    const start = Date.now();
+    const doc = await collection.findOne({ place_id: id});
+    const end = Date.now();
+    return end - start;
+  }
+
+  const findNearbyDocsForId = async (id) => {
+    const start = Date.now();
+    const doc = await collection.findOne({ place_id: id});
+    const nearby = await collection.find({ place_id: {$in:doc.nearby}}).toArray();
+    const end = Date.now();
+    // console.log(`Benchmark #${benchmarkNum}: found ${nearby.length} nearby docs for id ${id} in ${end-start} ms`);
+    return end - start;
+  }
+
+  const findNearbyDocsForIdsInRange = async (first, last) => {
+    const start = Date.now();
+    for (var x = first; x < last; x++) {
+      const doc = await collection.findOne({ place_id: x});
+      const nearby = await collection.find({ place_id: {$in:doc.nearby}}).toArray();  
+    }
+    const end = Date.now();
+    return end - start;
+  }
+
+  const findDocNTimes = async (id, n) => {
+    const start = Date.now();
+    for (var x = 0; x < n; x++) {
+      await collection.findOne({ place_id: id});
+    }
+    let end = Date.now();
+    return end - start;
+  }
+
+  const findDocsInRange = async (first, last) => {
+    const start = Date.now();
+    for (var x = first; x < last; x++) {
+      await collection.findOne({ place_id: x});
+    }
+    let end = Date.now();
+    return end - start;
+  }
+
+  const findDocsDivisibleBy500k = async() => {
+    const start = Date.now();
+    for (var x = 0; x < 10000000; x += 500000) {
+      await collection.findOne({ place_id: x});
+    }
+    let end = Date.now();
+    return end - start;
+  }
+
+  const benchmarkSuite = async () => {
+    // Benchmark1: Retrieve the info for the restaurant with 
+    // place_id / place_id 1234567 once.
+    let benchmark1 = await findDocForId(1234567);
```

Comment:
this may have been intentional -- but just a quick note in case, that if you test for the same hardcoded id when you run the tests, after you run the script once these will all likely be very quick because the database will have cached the results for that id; potentially worth looking into using a random id each time to avoid this, but also not necessarily worth investing time in given our other deliverables this week. just want to note for future reference!